vmaas/vulnerabilities.go

```diff
@@ -278,8 +316,17 @@ func repos2definitions(c *Cache, r *Request) map[DefinitionID]CpeID {
 		}
 	}
 
-	candidateDefinitions := make(map[DefinitionID]CpeID)
+	// Convert CPE map to sorted slice
```

Comment:
maybe we can make repoIDs, contentSetIDs, and cpeIDs slices (use current maps only to get unique values)
something like
```go
for _, cpe := range cpes {
	if !cpeIDs[cpe] {
		cpeIDs[cpe] = true
		uniqueCpeIDs = append(uniqueCpeIDs, cpe)
	}
}
```
then we won't need to have use `sort`, since we will be iterating only over slices and those slices which should be ordered if queries in cache load have `order by` (not sure if I didn't remove that 😄 )